src/Encryption/Codec/EncryptionCodec.cs

```diff
@@ -30,7 +30,6 @@ public EncryptionCodec(string keyID = DefaultKeyID, byte[]? key = null)
 
     public void Dispose()
```

Comment:
Can just remove this call, destructor, and `IDisposable` from this class I believe.